src/site/content/en/blog/indexeddb/index.md

```diff
@@ -228,196 +287,293 @@ All data operations in IndexedDB are carried out inside a transaction. Each oper
+As usual, you start by getting the database object, creating a transaction, and opening an object
+store. You call the `openCursor()` method on the object store and pass the cursor object to the
+callback function in `.then()`. This time you name the callback function `"logItems"` so you can
+call it from inside the function and make a loop. The line `if (!cursor) {return;}` breaks the loop
+if the promise returned by `store.openCursor()` resolves with `undefined`, or if the promise
+returned by `cursor.continue()`resolves with `undefined` (indicating that there are no more
+objects).
+
+The cursor object contains a `key` property that represents the primary key for the item. It also
+contains a `value` property that represents the data. At the end of `logItems()`, you return
+`cursor.continue().then(logItems)`. The `cursor.continue()` method returns a promise that resolves
+to a cursor object representing the next item in the store or `undefined` if there are no more
+objects. This result is passed to the callback function in `.then()`, which you have chosen to be
+`logItems()`, so that the function loops. Consequently, `logItems()` continues to call itself until
+no objects remain.
 
 ### Working with ranges and indexes {: #ranges }
 
-We can get all the data in a couple of different ways, but what if we want only a subset of the data based on a particular property? This is where indexes come in. Indexes let us fetch the data in an object store by a property other than the primary key. We can create an index on any property (which becomes the keypath for the index), specify a range on that property, and get the data within the range using the `getAll` method or a cursor.
+You can get all the data in a couple of different ways, but what if you want only a subset of the
+data based on a particular property? This is where indexes come in. Indexes let you fetch the data
+in an object store by a property other than the primary key. You can create an index on any property
+(which becomes the keypath for the index), specify a range on that property, and get the data within
+the range using the `getAll()` method or a cursor.
 
-We define the range using the `IDBKeyRange` object. This object has four methods that are used to define the limits of the range: `upperBound`, `lowerBound`, `bound` (which means both), and `only`. As expected, the `upperBound` and `lowerBound` methods specify the upper and lower limits of the range.
+You define the range using the `IDBKeyRange` object. This object has four methods that are used to
+define the limits of the range: `upperBound()`, `lowerBound()`, `bound()` (which means both), and
+`only()`. As expected, the `upperBound()` and `lowerBound()` methods specify the upper and lower
```

Comment:
`only()` is never defined.

Reply:
`includes()` was also missing. Adding links.